src/languages/en_US.json

```diff
@@ -3330,6 +3330,23 @@
         "to_claim_on": "to claim on %{network}",
         "view_coin": "View Coin",
         "you": "You"
+      },
+      "errors": {
+        "header": "Error launching token",
+        "api_request_failed": "API request failed",
+        "api_response_invalid": "API response invalid",
+        "contract_interaction_failed": "Contract interaction failed",
+        "gas_estimation_failed": "Gas estimation failed",
+        "insufficient_funds": "Insufficient funds",
+        "submission_details_missing": "Submission details missing",
+        "transaction_rejected": "Transaction rejected",
+        "transaction_failed": "Transaction failed",
```

Comment:
For the contract interaction, rejected, and failed errors, can we surface the contract error codes? Or are we limited by BE there?

Reply:
There would be a little more work involved in surfacing that, but I'm not blocked by BE. Let me know what the desired outcome is there(like is this about returning additional metadata or do we want the contract error codes incorporated into error strings) and I'll make a ticket.